README.md

```diff
@@ -34,19 +34,16 @@ API](https://developers.google.com/google-ads/api/docs/start).
     installed Composer is available on the command line as `composer`.
 *   System requirements and dependencies can be found in
     [composer.json](composer.json) of this library.
-    *   The minimum required PHP version of this library is the minimum PHP version for which the PHP development team still provide security fixes. Whenever such a version is sunset, we'll update the composer file accordingly. Currently, the update frequency is around once a year based on the [official schedule](https://www.php.net/supported-versions.php).
-    *   To install the gRPC PHP extension, follow the section
-        **Install the gRPC PHP extension**
-        of [this page](https://grpc.io/docs/quickstart/php.html).
+    *   **PHP**: You can find the required minimum PHP version in `"php"` of the [**require**](https://getcomposer.org/doc/01-basic-usage.md#the-require-key) key of `composer.json`. We usually set it to the minimum PHP version for which the PHP development team still provide security fixes. Whenever such a version is sunset, we'll update the composer file accordingly. Currently, the update frequency is around once a year based on the [official schedule](https://www.php.net/supported-versions.php). Visit [this page](https://www.php.net/manual/en/getting-started.php) for introduction to PHP.
+    *   **gRPC**: To install the gRPC PHP extension, visit the 
+        [**Install the gRPC PHP extension**](https://grpc.io/docs/languages/php/quickstart/#install-the-grpc-php-extension)         section. It takes some time to install.
```

Comment:
You could add a sentence at the end, something like: "To learn more about this requirement, please check the [`Transport guide`](https://developers.google.com/google-ads/api/docs/client-libs/php/transport)."